operators/pkg/instctrl/suite_test.go

```diff
@@ -58,7 +58,12 @@ var (
 	whiteListMap = map[string]string{"production": "true"}
 )
 
-const webdavSecretName = "webdav-secret"
+const (
+	webdavSecretName       = "webdav-secret"
+	NFSSecretName          = "user-pvc-secret"
+	NFSSecretServerNameKey = "server-name"
+	NFSSecretPathKey       = "path"
+)
```

Comment:
Constants shall not be duplicated.

Reply:
Removed duplicated constants, now they are defined only in tenant_controller